src/main/kotlin/no/fdk/dataset_catalog/controller/RDFController.kt

```diff
@@ -5,35 +5,26 @@ import no.fdk.dataset_catalog.service.RDFService
 import org.springframework.http.HttpStatus
 import org.springframework.http.ResponseEntity
 import org.springframework.web.bind.annotation.*
-import javax.servlet.http.HttpServletRequest
 
 @RestController
 @CrossOrigin
-@RequestMapping(
-    value = ["/catalogs"],
-    produces = ["text/turtle",
-        "application/rdf+json",
-        "application/ld+json",
-        "application/rdf+xml",
-        "application/n-triples"])
-class RDFController (val rdfService: RDFService) {
+@RequestMapping(value = ["/catalogs"], produces = ["text/turtle"])
```

Comment:
Det ser ut som om vi her fjerner støtte for alle andre standard RDF-serialiseringer enn turtle. Er det meininga?

Generelt sett bør vi tilby alle "standard" RDF-serialiseringer i alle tjenester.

Reply:
Tjenesten støtter ikke anna enn turtle atm, får 403 om du prøver, dette gir bare inntrykk av den gjør det. Bedre om du oppretter issue på at den skal støtte mer enn turtle

Reply:
403 var jo litt pussig. Burde ikkje det vere 406 i så fall?

Reply:
Nytt issue her: #84

Reply:
endepunktet støtter turtle og json, alt som ikke er turtle blir henvist til json-controller, som krever at du er pålogget. Så du får 406 om du er pålogget :man_shrugging: